common/src/main/java/com/ericsson/bss/cassandra/ecaudit/common/record/AuditOperation.java

```diff
@@ -28,4 +28,12 @@
      * @return the operation as a string
      */
     String getOperationString();
+
+    /**
+     * Provide the operation string for this operation without bound values being appended. This applies to
+     * prepared statement operations.
+     *
+     * @return the operation without bound values as a string
+     */
+    String getOperationStringWithoutBoundValues();
```

Comment:
Should we call this getNakedOperationString() so that we're using the same lingo in documentation and code?

Reply:
Sure, I named the parameter in the end and never changed this...